charts/galoy-auth/charts/auth-backend/values.yaml

```diff
@@ -0,0 +1,23 @@
+image:
+  repository: us.gcr.io/galoy-org/web-wallet
+  digest: "sha256:ab4f308adba7241ea4ab71bc520b7e4fa404ba86fc65f77bb9d37d0ad19b63fc"
+  git_ref: "c9df056" # Not used by helm
+ingress:
+  enabled: false
+service:
+  port: 80
+  type: ClusterIP
+authEndpoint: api/login
```

Comment:
I think this should be the whole url not just a path

Reply:
Not sure if this is being used, or will be used in auth backend. Will fix in a later PR if it's used.